spring-cloud/spring-cloud-stream/spring-cloud-stream-kafka/src/main/java/com/baeldung/schema/Employee.java

```diff
@@ -0,0 +1,517 @@
+/**
+ * Autogenerated by Avro
```

Comment:
We should not check in generated code - please remove and set up the .gitignore appropriately.

Reply:
done